DokuwikiXMLExport.php

```diff
@@ -85,9 +85,9 @@ private function getPageIds()
         $indexer = new Doku_Indexer();
         $pagesAndDeletedPages = $indexer->getPages();
 
-        // Get all pages that do have a description
+        // Get all pages that do have a description and a title set
         $pagesAndDeletedPages = array_filter($pagesAndDeletedPages, function ($page, $k) {
-            return (p_get_metadata($page)['description'] !== '' && p_get_metadata($page)['title'] !== '');
+            return (p_get_metadata($page)['description'] !== '' && !empty(p_get_metadata($page)['title']));
```

Comment:
Explain why you are checking description for `''` and title with `empty`. Probably extract both conditions into their own variables to make the statement clearer.

Reply:
Actually both should be tested for empty. Will add.